src/System.Net.Security/tests/FunctionalTests/System.Net.Security.Tests.csproj

```diff
@@ -6,7 +6,7 @@
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
     <ProjectGuid>{A55A2B9A-830F-4330-A0E7-02A9FB30ABD2}</ProjectGuid>
     <OutputType>Library</OutputType>
-    <UnsupportedPlatforms>Linux;OSX;FreeBSD</UnsupportedPlatforms>
+    <UnsupportedPlatforms>OSX;FreeBSD</UnsupportedPlatforms>
```

Comment:
couldn't verify this on OSX as don't have any OSX setup ( our request to access OSX lab machine is still pending) .. neither they run in CI by default .. Will follow up with Stephen to get a way to test this before enabling it .
I assume since the versions are not same , 100% success may not be the case ,so have to verify this..